projects/plugins/jetpack/package.json

```diff
@@ -168,6 +170,7 @@
 		"@wordpress/core-data": "2.12.3",
 		"@wordpress/dependency-extraction-webpack-plugin": "2.4.0",
 		"@wordpress/e2e-test-utils": "4.3.1",
+		"@wordpress/e2e-tests": "2.0.1",
```

Comment:
I wonder why do we need this dependency. I was not been able to figure what parts of it are used here.

Reply:
I was initially hoping to use some of the fixture utility code from the gutenberg test package but it wasn't exported in a way that was useful, but forgot to remove the dependency - gone now.